src/state/slices/txHistorySlice/txHistorySlice.ts

```diff
@@ -60,6 +64,9 @@ export type TxHistory = {
 }
 
 export type TxMessage = { payload: { message: Tx; accountSpecifier: string } }
+export type TxsMessage = {
```

Comment:
This will take precedence over the previous line which should be removed if that's the new format

Reply:
The new type is plural, so it's meant to take in multiple txs from getAllTxHistory whereas the first one is meant to handle a single transaction from the websocket.